Automattic/vip-go-mu-plugins
# VIP Feed Cache: Fix PHP 8 error non-static method VIP_Go_Feed_Cache::create() cannot be called statically

## Description
Calling `fetch_feed()` causes a PHP 8 fatal `non-static method VIP_Go_Feed_Cache::create() cannot be called statically`. Similar to https://core.trac.wordpress.org/changeset/49565, we can use register the `VIP_Go_Feed_Cache_Transient` class.

Note: After all sites have migrated to k8s, we can remove the entire VIP Go Feed Cache plugin since `mtime()` won't differ across pods.

## Changelog Description

### Plugin Updated: VIP Go Feed Cache

Fixes PHP 8 fatal "non-static method VIP_Go_Feed_Cache::create() cannot be called statically".

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) `$feed = fetch_feed('https://www.nasa.gov/rss/dyn/breaking_news.rss');`
2) Dump out `$feed` and take note of the build # (ctrl + F "build")
3) Apply patch
4) Repeat step 2 and ensure it matches up
5) Check Debug Monitor and notice that the error is gone
